specification/components/schemas/path/AppointmentId.yaml

```diff
@@ -0,0 +1,9 @@
+name: id
+in: path
+description: |
+    The ID of the appoinment to be supplied. It must be valid and relate to an existing Appointment Id otherwise a HTTP 404 response will be returned.
```

Comment:
> The ID of the appoinment to be supplied.
... to be updated.

> It must be valid and relate to an existing Appointment Id otherwise a HTTP 404 response will be returned.
.. to an existing Appointment Id otherwise...